dashboard/app/helpers/plc/evaluations_helper.rb

```diff
@@ -0,0 +1,34 @@
+module Plc::EvaluationsHelper
+  def determine_preferred_learning_modules level_source, script
```

Comment:
I don't understand - all the logic for determining whats the next thing to go to after completing a level was in script_levels_helper, not script_levels. This is following that.

This doesn't belong to enrollment module assignment (except as a static method) because its not dependent on properties of any module assignment, only plc learning modules. Similar point for unit assignments.